addons/l10n_ch/i18n_extra/de.po

```diff
@@ -69,73 +69,73 @@ msgstr "2.5% Inkl."
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25_purchase
-msgid "2.5% achat"
+msgid "2.5% purch."
 msgstr "2.5% Einkauf"
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25_purchase_incl
-msgid "2.5% achat Incl."
+msgid "2.5% purch. Incl."
 msgstr "2.5% Einkauf Inkl."
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25_invest
 msgid "2.5% invest."
-msgstr "2.5% Invest."
+msgstr "2.5% invest."
 
 #. module: l10n_ch
 #: model:account.tax.template,description:l10n_ch.vat_25_invest_incl
 msgid "2.5% invest. Incl."
-msgstr "2.5% Invest. Inkl."
+msgstr "2.5% invest. Inkl."
 
 #. module: l10n_ch
 #: model:account.tax.report.line,name:l10n_ch.account_tax_report_line_chtax_200
-msgid "200 Chiffre d'affaires"
+msgid "200 Total amount of agreed or collected consideration incl. from supplies opted for taxation, transfer of supplies acc. to the notification procedure and supplies provided abroad (worldwide turnover)"
```

Comment:
I had a few other notes that I lost due to a timeout :-) The most important one was a remark that some names become too long, and not really relevant. And I think not so user-friendly. E.g. here, I would stick with **Turnover**. thanks